config/ninja.php

```diff
@@ -84,11 +84,10 @@
         'username' => 'user@example.com',
         'clientname' => 'client@example.com',
         'password' => 'password',
-        'gocardless' => env('GOCARDLESS_KEYS',''),
-        'square' => env('SQUARE_KEYS',''),
-        'eway' => env('EWAY_KEYS',''),
-        'mollie', env('MOLLIE_KEYS',''),
-        'paytrace' => env('PAYTRACE_KEYS',''),
+        'gocardless' => env('GOCARDLESS_KEYS', ''),
+        'square' => env('SQUARE_KEYS', ''),
+        'eway' => env('EWAY_KEYS', ''),
+        'paytrace' => env('PAYTRACE_KEYS', ''),
```

Comment:
I think this got removed accidentally, could you add back please?

        'mollie', env('MOLLIE_KEYS',''),

Reply:
sorry for that. resolved

Reply:
oh sorry, it is down below on line 102

Reply:
i saw, that you have 'mollie', env('MOLLIE_KEYS','') onstead of 'mollie' => env('MOLLIE_KEYS','') in your repo. which one is the right one?

Reply:
'mollie', env('MOLLIE_KEYS',''),